docs/perf/perf_tuning.rst

```diff
@@ -40,6 +42,24 @@ Below are key factors for tuning vLLM-based rollout. Before tuning, we recommend
 More tuning details such as dealing with Preemption and Chunked-prefill
 can be found in `vLLM official tuning guide <https://docs.vllm.ai/en/latest/performance/optimization.html>`_ 
 
+Enable remove padding (sequence packing)
+-----------------------------------------
+
+Currently, for llama, mistral, gemma and qwen based models, users can enable `use_remove_padding=True` to utilize the 
```

Comment:
Do we really support Gemma?

Reply:
Yes, Gemma1 is tested. Gemma2 didn't support rmapd

Reply:
Could you specifically add Gemma 1 to clear confusion?